src/test/test.ts

```diff
@@ -0,0 +1,124 @@
+
+import * as assert from "assert";
+import * as fs from "fs";
+import * as mocha from "mocha";
+import * as officeToolbox from "../util";
+import * as path from "path";
+import * as util from "util";
+const application = "excel";
+
+describe("Add manifest", function () {
+    describe(process.platform === "win32" ? "Add manifest to the registry" : "Add the manifest to the WEF folder", function () {
+        this.timeout(0);
+        let manifestFound = false;
+        let manifests: string[];
+        let testManifestPath = path.resolve(`${process.cwd()}/src/test/test-manifest.xml`);
```

Comment:
Should the "test" folder be moved so it is outside the "src/" folder?

Reply:
I was trying to do that initally and was hittinga ts-node error when running the tests: "TypeError: test/test.ts Emit skipped. I will do a bit more research on how to work around this error and will move the test folder under root if/when I have a fix

Reply:
Fixed issue by removing rootDir setting from tsconfig.json and moved test folder to root